src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -15,6 +16,7 @@
     @Size(min = 8, max = 40)
     private String password;
     private String repeatPassword;
+    private Role role;
```

Comment:
let's remove it
when registering a new User, you already set the default USER role for him